hunting-buddy.lic

```diff
@@ -184,6 +184,8 @@ class HuntingBuddy
   end
 
   def check_prehunt_buffs
+    return unless @settings.waggle_sets['prehunt_buffs']
```

Comment:
Good add. The prior change to this script surfaced folks using prehunt buffs in ways we didn't anticipate, like https://discord.com/channels/745675889622384681/745696628714897508/819328371347161109